### ISSUE TITLE
"will-attach-webview" is never triggered

### ISSUE THREAD

**Issue by fabiospampinato** [electron/electron#24439]
### Preflight Checklist

* [ ] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ ] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 9.0.5
* **Operating System:**
  * macOS 10.14
* **Last Known Working Electron version:**
  * N/A

### Expected Behavior

"will-attach-webview" should be triggered.

### Actual Behavior

It's not triggered, making it impossible to implement [one the suggested steps](https://www.electronjs.org/docs/tutorial/security#1-only-load-secure-content) for securing electron apps.

### To Reproduce

Gist: https://gist.github.com/d1c69ddc270ffd9902b48150bc5fec2a

**Comment by pushkin-** [comment]
the `webviewTag` property of the `BrowserWindow`'s `webPreferences` is set to false by default, which is likely why you observe this behavior.

Try setting it to `true` and then creating you webview and attaching it.

**Comment by fabiospampinato** [comment]
I think I just assumed this would work for `BrowserView` as well other than the `<webview>` tag 🤔 Shouldn't it though? Maybe there should be a `will-attach-browserview` as well? 🤔

**Comment by pushkin-** [comment]
No, I don't think that will work for browser views. The `webcontents-created` I believe should fire for browser views. Are you trying to prevent attaching a browser view, or something?

**Comment by MarshallOfSound** [comment]
`BrowserView` and `webview` are fundamentally different, the webview events will not be fired for browserviews
